### PR TITLE
Fixes #21641 - Drop jquery-ui spinners in favor of React implementation

### PR DESCRIPTION
**Pull request by yifatmakias**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
The failures are definitelly not relevant, probably some npm package update?
I'll investigate next week if noone would get to it sooner.

**Comment by MariaAga** [comment]
This shouldn't be merged before it's usage is removed in plugins:
https://github.com/search?q=org%3Atheforeman+numFields&type=code
Also the deprecation says that it will be only removed in foreman 3.0

**Comment by ezr-ondrej** [comment]
> This shouldn'd be merged before it's usage is removed in plugins:
> https://github.com/search?q=org%3Atheforeman+numFields&type=code
> Also the deprecation says that it will be only removed in foreman 3.0

there is multiple levels to answer this.

1. 2.6 = 3.0 I'm just working on writup for it
2. plugins had time to do that, we should not be as concerned with plugins using some deprecated stuff
3. to ease the point 2, because we gave just one version to plugins to deal with this, the function calls can probably be just removed, as we have changed the helpers implementation to react implementation, so the initializers in the plugin just dosn't do anything

This being sad, we could leave blank function with the deprecation warning only for one more version, so the plugins don't get js errors just by calling useless function.
**EDIT**: Though I would just prefer plugin maintainers to remove the calls and also by that verify the new implementation is working for them. It is simple removal and simple test, that IMHO is not too much to ask from plugin maintainers if they wish to keep the plugins alive.

**Comment by ekohl** [comment]
If it's a simple migration, it can be beneficial (and faster) to submit a PR yourself.

File: webpack/assets/javascripts/bundle.js [hunk 1]
```diff
@@ -46,7 +45,6 @@ window.tfm = Object.assign(window.tfm || {}, {
   hosts,
   httpProxies,
   toastNotifications,
-  numFields,
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/bundle.js:49]: I can imagine having in this file
```js
const numFieldsDeprecationOnly = {
  initAll: () => { window.tfm.tools.deprecate('initAll()', 'does nothing as of now, please stop calling it',    '3.2') };
};
```
And then here: `numFields: numFieldsDeprecationOnly,` Which would prevent plugins from failing hard on the function missing for two more version.

**Review by ezr-ondrej — COMMENTED** [review]
Nice work!
Just noticed we are leaving some bits behind, can we drop these requires? :)
- https://github.com/theforeman/foreman/blob/dcf11351d798bc59886fded3e0dfbff01ea98482/app/assets/javascripts/application.js#L1
- https://github.com/theforeman/foreman/blob/dcf11351d798bc59886fded3e0dfbff01ea98482/app/assets/stylesheets/application.scss#L5-L6
- [whole file] https://github.com/theforeman/foreman/blob/dcf11351d798bc59886fded3e0dfbff01ea98482/app/assets/stylesheets/jquery.ui.spinner_custom.scss

**Review by ezr-ondrej — APPROVED** [review]
Thanks @yifatmakias ! Great drop :+1:

**Pull request merged**